File: twitter.php
<?php
session_start();
require_once("twitteroauth/twitteroauth/twitteroauth.php"); //Path to twitteroauth library
 
$twitteruser = "aaaronlopez";
$notweets = 30;
$consumerkey = "IHO7c3kX9cPXeC5yymwkX7ySF";
$consumersecret = "5WxVESOsFbF87KviFjn8S1oROjHrfmOLNMKfH0gbMgNAAkwU69";
$accesstoken = "2241581574-oaHtX7XWAjk2Ljv7TAobCCry3l5sW9ZUrKqGuWf";
$accesstokensecret = "aDjl7gNSIGIKjwumv2XqGtJXIHsC5WepRwhHVCBG8UKqP";
 
function getConnectionWithAccessToken($cons_key, $cons_secret, $oauth_token, $oauth_token_secret) {
  $connection = new TwitterOAuth($cons_key, $cons_secret, $oauth_token, $oauth_token_secret);
  return $connection;
}
 
$connection = getConnectionWithAccessToken($consumerkey, $consumersecret, $accesstoken, $accesstokensecret);
 
$tweets = $connection->get("https://api.twitter.com/1.1/statuses/user_timeline.json?screen_name=".$twitteruser."&count=".$notweets);
 
echo json_encode($tweets);
?>

File: twitter.py
from tweepy import OAuthHandler, Stream
from tweepy.streaming import StreamListener


consumer_key="IHO7c3kX9cPXeC5yymwkX7ySF"
consumer_secret="5WxVESOsFbF87KviFjn8S1oROjHrfmOLNMKfH0gbMgNAAkwU69"

access_token = "2241581574-oaHtX7XWAjk2Ljv7TAobCCry3l5sW9ZUrKqGuWf"
access_token_secret = "aDjl7gNSIGIKjwumv2XqGtJXIHsC5WepRwhHVCBG8UKqP"

auth = OAuthHandler(consumer_key, consumer_secret)
auth.secure = True
auth.set_access_token(access_token, access_token_secret)

# api = tweepy.API(auth)

import json

tweets = []
class listener(StreamListener):
	def __init__(self, api=None):
		super(listener, self).__init__()
		self.num_tweets = 0

	def on_data(self, data):
		if self.num_tweets < 3:
			d = json.loads(data)
			# print d.get('text')
			tweets.append(d.get('text'))
			self.num_tweets += 1
			print data
			return True
		else:
			return False

	def on_error(self, status):
		print status

twitterStream = Stream(auth, listener())
twitterStream.filter(track=["car"], languages=["en"])

print tweets

from textblob import TextBlob

#blob = TextBlob()

